Lib/tkinter/simpledialog.py

```diff
@@ -260,7 +260,15 @@ def __init__(self, title, prompt,
                  parent = None):
 
         if not parent:
-            parent = tkinter._default_root
+            if tkinter._default_root:
```

Comment:
I would recommend changing this to an explicit check for "not None"

Reply:
Yes. The only reason why I did not do it, is to make as little change as possible.